bib/internal/source/source.go

```diff
@@ -58,6 +61,36 @@ func uefiVendor(root string) (string, error) {
 	return "", fmt.Errorf("cannot find UEFI vendor in %s", bootupdEfiDir)
 }
 
+func readSelinuxPolicy(root string) (string, error) {
```

Comment:
This is fine by me. That said my longer term preference is that bootc is responsible for labeling files at least in the simple cases, not the installer. I took a step in this direction with https://github.com/containers/bootc/pull/267 but needs some design thought.

i.e. in the general case the disk image generator passes a tarball or maybe a `FROM scratch` OCI image it doesn't write files directly. Then it's always bootc that owns labeling.

Reply:
In b-i-b we also need to relabel the files in the build pipeline, so even that may not be enough to replace the need for this change.

Reply:
> In b-i-b we also need to relabel the files in the build pipeline

Why? 

The big exception of course is the anaconda ISO case which just to keep repeating myself intersects with https://github.com/rhinstaller/anaconda/discussions/5888 because then we'd make the Anaconda ISO actually built as a container and using bootc, which would make everything symmetric.

Reply:
It's not _strictly_ necessary, but we get selinux warnings otherwise.  See The reason for these labels is explained here: https://github.com/osbuild/images/commit/dea1af482741dd57c3dfc44024e9adba1c6390c6